Britannia/common/scripted_triggers/BTWK_cultural_triggers.txt

```diff
@@ -178,4 +178,15 @@ is_imperial_like_culture_trigger = {
 			is_romanized_germanic_culture_trigger = yes
 		}
 	}
+}
+
+culture_is_imperial_like_culture_trigger = {
```

Comment:
Shouldn't be independent of the is_imperial_like_culture_trigger 
Alternatively, do you actually need this trigger right now?

Reply:
Nope, just about to remove it